api/client/util.ts

```diff
@@ -0,0 +1,52 @@
+import { camelCase, snakeCase, mapKeys, mapValues, cloneDeep, isPlainObject, isArray, map, functions } from "lodash";
+import axios, { Axios, AxiosInstance, AxiosRequestConfig, AxiosResponse } from "axios";
+
+/**
+ * Deeply convert keys within an object to a given case (either snake_case or camelCase are
+ * supported).
+ */
+function keysToCase(object: any, method: typeof camelCase | typeof snakeCase): any {
+  let convertedObject = cloneDeep(object);
+  if (isArray(convertedObject)) {
+    return map(convertedObject, (obj) => keysToCase(obj, method));
```

Comment:
Since this function is called recursively, when it's not an object, it'll hit the `else` below and the final `return` that just returns the value itself.